SingularityUI/app/templates/taskDetail/taskOverview.hbs

```diff
@@ -1,8 +1,29 @@
+{{#if data.isStillRunning}}
+    {{#ifNotEqual data.killStatus 'kill9'}}
+        {{#if data.isStillRunning}}
+            {{#if data.isInCleanup}} 
+                <div class="alert alert-warning" role="alert">Request is bouncing. This task will be killed once all replacement tasks are healthy. You may still force kill this task.</div>
```

Comment:
Are you referencing the `data.isInCleanup`? That's being set from a request to `tasks/cleaning` which I thought was the `SingularityTaskCleanup` object. Or are you referencing the `data.isStillRunning`?